files/enc.rb

```diff
@@ -202,7 +202,7 @@ def build_body(certname,filename)
 
   begin
     require 'facter'
-    puppet_facts['values']['puppetmaster_fqdn'] = Facter.value(:fqdn).to_s
+    puppet_facts['values']['puppetmaster_fqdn'] = Facter.value('networking.fqdn').to_s
```

Comment:
I also don't see a benefit in the `.to_s`, but it was there before, so we should probably keep it.